projects/components/src/utils/test/locator/cypress-widget-locator.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { IdGenerator } from '../../id-generator/id-generator';
+import { FindableWidget, FindParams, isFindParamsObject } from '../widget-object';
+import { BaseWidgetLocator, CorrectReturnTypes, FindableLocator, LocatorDriver } from './widget-locator';
+
+type Chainable = Cypress.Chainable<JQuery<HTMLElement>>;
+
+// TODO: use alias to find widgets within other widgets
+/**
+ * Knows how to find Cypress chainables in the DOM.
+ */
+export class CypressLocatorDriver implements LocatorDriver<Chainable> {
```

Comment:
There's no need for driver in the name, it's just a locator. Driver was when it was responsible for driving the test (clicking, typing)

Reply:
I think based on our conversation we decided to keep this name the same, but let me know if I interpreted that